test/test_bundle.js

```diff
@@ -2927,12 +2927,12 @@ describe('json-pollock tests', function () {
     });
 
     it('carousel should move to the right', function (done) {
-      chai.expect(window.getComputedStyle(carouselListRoot).left).to.be.equal('0px');
+      const prevLeft = window.getComputedStyle(carouselListRoot).left;
       carouselRight.click();
 
       setTimeout(function () {
-        const left = parseInt(window.getComputedStyle(carouselListRoot).left);
-        chai.expect(left).to.not.equal(0);
+        const left = window.getComputedStyle(carouselListRoot).left;
+        chai.expect(left).to.not.equal(prevLeft);
```

Comment:
I think it is better to check that `left` is higher/lower than `prevLeft` according to the expected flow

Reply:
@meirotstein done, but it will be `.to.be.below` for both cases